### ISSUE TITLE
Alphabetical Sort is case-sensitive

### ISSUE THREAD

**Issue by BernsteinA** [Hirse/brackets-outline-list#48]
i.e. when sort is enabled, all functions that start with capital letters go above functions that start with lower case letters

**Comment by BernsteinA** [comment]
https://github.com/Hirse/brackets-outline-list/pull/49

**Comment by Hirse** [comment]
I personally don't see this as an issue for most languages.
Although it could be changed for CSS and Markdown I guess.

**Comment by BernsteinA** [comment]
Not sure why it should differ per language. Was it an intentional choice?

**Comment by Hirse** [comment]
Yes, this way Constructor Functions in JS are above others, which makes sense to me.
